dashboard/test/ui/features/step_definitions/eyes_steps.rb

```diff
@@ -44,7 +48,11 @@
 
   @browser = @original_browser
   fail_on_mismatch = !CDO.ignore_eyes_mismatches
-  @eyes.close(fail_on_mismatch)
+  begin
+    @eyes.close(fail_on_mismatch)
+  rescue Applitools::TestFailedError => e
+    puts "<span style=\"color: red;\">#{EYES_ERROR_PREFIX} #{linkify(e)}</span>"
```

Comment:
Which log does this puts go to?

Reply:
This html ends up in the cucumber log output, e.g. including:
1. the html output (e.g. ChromeLatestWin7_angleHelper_eyes_output.html), and
2. the stdout stream (which is consumed by runner.rb, and may also what you'd end up with in `success.log` if you didn't specify `runner.rb --html`).